Installation Failure

The "**get**" command is deprecated as of 1.17 last August. 

`https://go.dev/doc/go-get-install-deprecation#what-to-use-instead`

The alternative "**install**" command using emitter@latest fails here:

```
# [github.com/emitter-io/emitter/config](http://github.com/emitter-io/emitter/config)
go/pkg/mod/github.com/emitter-io/emitter@v2.0.437+incompatible/config/config.go:66:18: undefined: config.VaultConfig
go/pkg/mod/github.com/emitter-io/emitter@v2.0.437+incompatible/config/config.go:75:31: undefined: config.VaultConfig
go/pkg/mod/github.com/emitter-io/emitter@v2.0.437+incompatible/config/config.go:84:21: undefined: config.NewVaultCache

```
Downloading the repo and running go **build** fails with security warning: 

```
verifying [github.com/google/flatbuffers@v1.12.0](http://github.com/google/flatbuffers@v1.12.0): checksum mismatch
	downloaded: h1:N8EguYFm2wwdpoNcpchQY0tPs85vOJkboFb2dPxmixo=
	go.sum:   h1:/PtAHvnBY4Kqnx/xCQ3OIV9uYcSFGScBsWI3Oogeh6w=
```

--- a/README.md
+++ b/README.md
@@ -59,7 +59,7 @@ Notice: You must use -e for docker environment.You could get it from your docker
 docker run -d --name emitter -p 8080:8080 --restart=unless-stopped emitter/server
 ```
 
-Alternatively, you build and run from source. 
+Alternatively, you can build and run from source. 
 
 ```shell
 # install git and go
@@ -68,7 +68,6 @@ git clone https://github.com/emitter-io/emitter
 cd emitter
 go get -x .
 go build -x .
-go install -x .
 ./emitter
 ```
 

--- a/README.md
+++ b/README.md
@@ -59,10 +59,17 @@ Notice: You must use -e for docker environment.You could get it from your docker
 docker run -d --name emitter -p 8080:8080 --restart=unless-stopped emitter/server
 ```
 
-Alternatively, you might compile this repository and use `go get` command to rebuild and run from source. 
+Alternatively, you build and run from source. 
 
 ```shell
-go get -u github.com/emitter-io/emitter && emitter
+# install git and go
+# depending on your system, you may also need to install gcc and musl-dev
+git clone https://github.com/emitter-io/emitter
+cd emitter
+go get -x .
+go build -x .
+go install -x .
+./emitter
 ```
 
 ### Get License

